CRM/Contribute/Form/ManagePremiums.php

```diff
@@ -209,16 +209,14 @@ public function buildQuickForm() {
   public static function formRule($params, $files) {
 
     // If choosing to upload an image, then an image must be provided
-    if (
-      isset($params['imageOption']) &&
-      $params['imageOption'] == 'image' &&
-      empty($files['uploadFile']['name'])
+    if (CRM_Utils_Array::value('imageOption', $params['imageOption']) == 'image'
```

Comment:
The code: 
`CRM_Utils_Array::value('imageOption', $params['imageOption'])` 
needs to  be changed to
`CRM_Utils_Array::value('imageOption', $params)`